packages/core/src/base-chart.ts

```diff
@@ -954,4 +980,30 @@ export class BaseChart {
 	getBBox(selector: any) {
 		return this.innerWrap.select(selector).node().getBBox();
 	}
+// ================================================================================
+	// Makes chart data components more screen reader friendly
+	// ================================================================================
+	makeAccessible (element?: any, dataList?: any) {
+		// Make chart data components tabbable
+		element.attr("tabindex", 0);
+		// Aria label contains the x value, y value, and dataset for the data element
+		if (element.attr("class") === "bar" || element.attr("class") === "dot") {
```

Comment:
What if we decide to change the `class` for bars or dots down the road? We need a better way here to distinguish the elements